src/visit_vtk/lightweight/vtkDebugStream.h

```diff
@@ -22,12 +22,12 @@ class VISIT_VTK_LIGHT_API vtkDebugStream : public vtkOutputWindow
   static void   FrequentEventsFilter(bool on){filterFrequentEvents = on;}
 
 protected:
-  vtkDebugStream() {};
-  virtual ~vtkDebugStream() {};
+  vtkDebugStream() {}
+  virtual ~vtkDebugStream() {}
 
 private:
-  vtkDebugStream(const vtkDebugStream&); 
-  void operator=(const vtkDebugStream&);
+  vtkDebugStream(const vtkDebugStream&) = delete; 
+  void operator=(const vtkDebugStream&) = delete;
```

Comment:
what warning suggested deleting the methods?

Reply:
Actually none.  Sorry about that. I may have gotten carried away. Most vtk files utilize this construct and I was looking at a lot of files in vtk proper to get insight into the logic in some of the files I was changing. I can revert all the additions of '= delete' since it is so controversial.

Reply:
I'm not necessarily against it, I just wanted to make sure that we were deleting the functions for good reason. If deleting them or leaving them unimplemented has no real difference on our lives, then I don't want to make you go back and change everything. I just wanted to understand the rationale and make sure we were doing the right thing.